tdrs-backend/tdpservice/email/email.py

```diff
@@ -6,61 +6,96 @@
 from django.core.mail import EmailMultiAlternatives
 from django.conf import settings
 from django.template.loader import get_template
+from django.contrib.admin.models import LogEntry, ContentType, CHANGE
 
 import logging
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger()
 
 
-@shared_task
-def mail(email_path, recipient_email, email_context):
-    """Send an automated email to a user. Use mail.delay() to send asynchronously."""
-    subject = email_context["subject"]
-    html_message = construct_email(email_path, email_context)
+def log(msg, logger_context=None, level='info'):
+    """Create a log in the terminal and django admin console, for email tasks."""
+    from tdpservice.users.models import User
+
+    log_func = logger.info
+
+    match level:
+        case 'info':
+            log_func = logger.info
+        case 'warn':
+            log_func = logger.warn
+        case 'error':
+            log_func = logger.error
+        case 'critical':
+            log_func = logger.critical
+        case 'exception':
+            log_func = logger.exception
+
+    log_func(msg)
+
+    if logger_context:
```

Comment:
🚀 🚀 🚀 thanks for this @jtimpe! 

@ttran-hub email logs from django below:
![1814p1](https://user-images.githubusercontent.com/63075587/197044083-6a7cd383-be7e-4e14-b2bc-bcef2af29d27.PNG)